astroquery/esa/jwst/tests/test_jwsttap.py

```diff
@@ -953,57 +953,61 @@ def test_query_target(self):
 
         with pytest.raises(ValueError) as err:
             jwst.query_target(target_name="M1", target_resolver="NED",
                               radius=units.Quantity(5, units.deg))
-        assert coordinate_error in err.value.args[0]
+        assert 'This target name cannot be determined with this resolver: NED' in err.value.args[0]
 
         with pytest.raises(ValueError) as err:
             jwst.query_target(target_name="M1", target_resolver="VIZIER",
                               radius=units.Quantity(5, units.deg))
-        assert coordinate_error in err.value.args[0]
-
-        # Testing valid coordinates from resolvers
-        dummyTapHandler = DummyTapHandler()
-        jwst = JwstClass(tap_plus_handler=dummyTapHandler)
-        # default parameters
-        parameters = {}
-        parameters['name'] = None
-        parameters['output_file'] = None
-        parameters['output_format'] = 'votable'
-        parameters['verbose'] = False
-        parameters['dump_to_file'] = False
-        parameters['upload_resource'] = None
-        parameters['upload_table_name'] = None
-        simbad_file = data_path('test_query_by_target_name_simbad.vot')
-        simbad_table = Table.read(simbad_file)
-        simbad.query_object = MagicMock(return_value=simbad_table)
-        ned_file = data_path('test_query_by_target_name_ned.vot')
-        ned_table = Table.read(ned_file)
-        ned.query_object = MagicMock(return_value=ned_table)
-        vizier_file = data_path('test_query_by_target_name_vizier.vot')
-        vizier_table = Table.read(vizier_file)
-        vizier_table_list = TableList({'1': vizier_table})
-        vizier.query_object = MagicMock(return_value=vizier_table_list)
-
-        with open(data_path('test_query_by_target_name_simbad_query.txt'), 'r') as file:
-            parameters['query'] = file.read()
-            jwst.query_target(target_name="M1", target_resolver="SIMBAD",
-                              radius=units.Quantity(5, units.deg))
-            dummyTapHandler.check_call('launch_job', parameters)
-        with open(data_path('test_query_by_target_name_ned_query.txt'), 'r') as file:
-            parameters['query'] = file.read()
-            jwst.query_target(target_name="M1", target_resolver="NED",
-                              radius=units.Quantity(5, units.deg))
-            dummyTapHandler.check_call('launch_job', parameters)
-        with open(data_path('test_query_by_target_name_vizier_query.txt'), 'r') as file:
-            parameters['query'] = file.read()
-            jwst.query_target(target_name="M1", target_resolver="VIZIER",
-                              radius=units.Quantity(5, units.deg))
-            dummyTapHandler.check_call('launch_job', parameters)
+        assert 'This target name cannot be determined with this resolver: VIZIER' in err.value.args[0]
+
+    # def test_query_target(self):
```

Comment:
Please do rather delete it than comment out, unless we're putting this back in once the remote tests are added with the functional archive

Reply:
Ok, I am going to remove it.